src/Emogrifier/HtmlProcessor/HtmlPruner.php

```diff
@@ -41,4 +41,47 @@ public function removeInvisibleNodes()
 
         return $this;
     }
+
+    /**
```

Comment:
By the way, would you like to add yourself as an `@author` to the class and the test case?

Reply:
Sure, will do 😀